staff/pedro-garriganogues/gamma-template/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <header class="header">
+        <h1 class="green-font">Gamma</h1>
+        <img class="burger" src="./images/bars-solid.svg" alt="404">
+        <section class="redirect">
+            <a class="link" href="index.html">Home</a>
+            <a class="link" href="about.html">About</a>
+            <a class="link" href="contact.html">Contact</a>
+        </section>
+    </header>
+    <section class="black-top">
+        <div>
+        <div class="title">
```

Comment:
Pay attention to the indentation, because without it it's more difficult to read the code.